kubernetes-secrets.tftpl

```diff
@@ -0,0 +1,58 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: spacelift-shared
```

Comment:
Maybe we can make this a var with a default value for secrets name. And export them as output in the shell env.
So layer we can reference them from here in the helm chart?

Just a random idea, feel free to ignore.

Reply:
I think I'll leave this one if it's ok, and we can do that if required later. I just want to keep it simple for now and avoid making more changes until we need to.

Reply:
Of course, that was just a random idea in case it could have been interesting. I'm not feeling strong at all about it 👍🏻